testing/model-mesh/7-analyze.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o errtrace
```

Comment:
I think all of those should be removed in this step, maybe except `set -o nounset`, so that the script captures as much as possible
```suggestion
set -o nounset
```

Reply:
Fixed